src/extension.ts

```diff
@@ -235,6 +235,76 @@ export function activate(context: vscode.ExtensionContext) {
     async () => await blueprintPreview(context.extensionPath)
   );
 
+  const exportBlueprint = vscode.commands.registerCommand(
+    "extension.exportBlueprint",
+    async () => {
```

Comment:
This method has many nested guard conditions, maybe it would be better to use a return statement each time a guard condition fails:
```js
if(<GuardCondition>){
   <ShowMessage>;
   return;
}
```